### ISSUE TITLE
4.3.0 Error missing package jiti

### ISSUE THREAD

**Issue by datyin** [antfu/eslint-config#668]
### Describe the bug

When manually installing latest version on fresh project ESLint crashes.

ESLINT Output:
```log
[Error - 5:04:35 PM] An unexpected error occurred:

[Error - 5:04:35 PM] Error: The "jiti" library is required for loading TypeScript configuration files. Make sure to install it.

```

Fix:
```node
pnpm add -D jiti
```

### Reproduction

When you do a fresh install of the package linting is not working because of the missing package.

### System Info

```Shell
System:
    OS: Windows 11 10.0.26100
    CPU: (16) x64 AMD Ryzen 7 7800X3D 8-Core Processor
    Memory: 44.66 GB / 63.66 GB
  Binaries:
    Node: 18.20.3 - C:\Program Files\nodejs\node.EXE
    npm: 10.9.2 - C:\Program Files\nodejs\npm.CMD
    pnpm: 10.4.0 - C:\Program Files\nodejs\pnpm.CMD
    bun: 1.1.2 - ~\.bun\bin\bun.EXE
  Browsers:
    Edge: Chromium (127.0.2651.98)
```

### Used Package Manager

pnpm

### Validations

- [x] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [x] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by antfu** [comment]
We temporarily closed this due to the lack of enough information. We could not identify whether it was a bug or a userland misconfiguration with the given info.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)
